pkg/configuration/get_configuration.go

```diff
@@ -0,0 +1,45 @@
+package configuration
+
+import "time"
+
+func GetBool(value *bool, defaultValue bool) bool {
+	if value != nil {
+		return *value
+	}
+	return defaultValue
+}
+
+func GetInt(value *int, defaultValue int) int {
+	if value != nil {
+		return *value
+	}
+	return defaultValue
+}
+
+func GetString(value *string, defaultValue string) string {
+	if value != nil {
+		return *value
+	}
+	return defaultValue
+}
+
+func GetDuration(value *string, defaultValue time.Duration) time.Duration {
```

Comment:
So, we are ignoring the case when `value` is not nil but invalid? Let's add a comment stating that the default value is returned when `value` is nil or invalid and can't be parsed as a duration.

Reply:
Done in 689af28e98b281b7dc11a4f88dfd23249163e572